src/main/java/com/miquido/validoctor/rule/Rules.java

```diff
@@ -191,6 +191,16 @@ private Rules() {
     return new SimpleRule<>("STRING_EXACT_LENGTH:" + exactLength, params, str -> str == null || str.length() == exactLength);
   }
 
+  /**
+   * Passed: patient is null or string that contains specified {@code text}.<br/>
+   * Violated: patient is string not containing specified {@code text}.
+   */
+  public static Rule<String> stringContains(String text) {
+    Map<String, Object> params = new HashMap<>();
+    params.put(TEXT, text);
+    return new SimpleRule<>("STRING_CONTAINS:" + text, params, str -> str == null || str.contains(text));
+  }
```

Comment:
Did you consider adding any pattern matching rules for Strings?
Probably useful for date format validation

Reply:
i did. I'll add it to this PR